imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -71,6 +71,7 @@ class CaseExplorer extends Component {
       })
     }
   }
+
```

Comment:
The added empty lines on this file should not be committed, as it just adds the file to the changed files list unnecessarily.